datadog/resource_datadog_synthetics_test_.go

```diff
@@ -1504,7 +1504,7 @@ func buildAssertions(attr []interface{}) []datadogV1.SyntheticsAssertion {
 							case
 								datadogV1.SYNTHETICSASSERTIONOPERATOR_LESS_THAN,
 								datadogV1.SYNTHETICSASSERTIONOPERATOR_MORE_THAN:
-								if match, _ := regexp.MatchString("{{\\s*[a-zA-Z0-9]+\\s*}}", v.(string)); match {
+								if match, _ := regexp.MatchString("{{\\s*([^{}]*?)\\s*}}", v.(string)); match {
```

Comment:
@AlaricCalmette why `*?` instead of `+`?
```suggestion
								if match, _ := regexp.MatchString("{{\\s*([^{}]+)\\s*}}", v.(string)); match {
```

Reply:
To be consistent with what we check in web-ui [here](https://github.com/DataDog/web-ui/blob/preprod/javascript/datadog/synthetics/lib/constants.ts#L3)